mne/time_frequency/tests/test_psd.py

```diff
@@ -52,6 +53,124 @@ def test_psd():
 
 
 @requires_version('scipy', '0.12')
+def test_psd_welch():
```

Comment:
I see warnings when I run the tests. You should catch the warnings. Look at some other tests how we do it.